src/NotificationBar/NotificationBar.stories.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { Story, Meta } from '@storybook/react';
+import { NotificationBar, NotificationBarProps } from './NotificationBar';
+import { StyledWarningIcon } from './NotificationBar.styles';
+
+export default {
+  title: 'Components/NotificationBar',
+  component: NotificationBar,
+} as Meta;
+
+const Template: Story<NotificationBarProps> = args => (
+  <NotificationBar {...args}>
+    <div style={{ display: 'flex', alignItems: 'center' }}>
+      <StyledWarningIcon />
+      <span>
+        You are currently using an outdated browser, this may cause issues using our site. Upgrade to a modern browser
+        for a better experience
+      </span>
+    </div>
+  </NotificationBar>
+);
+
+export const Controlled = Template.bind({});
```

Comment:
What is the bind doing here? Can't see it in the docs just.

Reply:
This references it: https://storybook.js.org/docs/react/essentials/controls

Basically the Template above documents how the props are passed to the component and the bind makes each story use its own object. We can then use 
`Controlled.args = {
...
}
`
to help change how the story looks.

I used template and bind here so that in the future if we want to add more example stories then its super easy and can just reuse that rather than copy pasting